src/api.yaml

```diff
@@ -0,0 +1,127 @@
+constant:
+  array layer count undefined: Undefined value for array layer count
+  copy stride undefined: Undefined value for copy stride
+  limit u32 undefined: Undefined U32 limit value
+  limit u64 undefined: Undefined U64 limit value
+  mip level count undefined: Undefined MIP level count
```

Comment:
nit: I don't think MIP is an acronym? I've never seen it capitalized (except on the Wikipedia page when looking it up just now. Weird)

Reply:
Apparently:

> The origin of the term mipmap is an initialism of the Latin phrase multum in parvo ("much in a small space"), and map, modeled on bitmap. 

But, will lowercase to be consistent with other places.